migration/sql-files/002-tracker-items.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE ONLY tracker_items
```

Comment:
> No, I haven't run old code with new schema. Is that required?

The way we deploy to production is:

* Start new version
** Migrate new version
** All ok? 
*** Stop old version

If new version change a shared resource between old and new in a way that old can't use it anymore, then old is broken until new is completely rolled out, if it ever is(might fail on other things). Similarly, if new fail because old is writing to the same resource, then we run into the chance that new never rolls out.